src/features/emails/l10n/messageIds.ts

```diff
@@ -165,4 +165,15 @@ export default makeMessages('feat.emails', {
     title: m('Targets'),
     viewButton: m('View target group'),
   },
+  unsubscribePage: {
+    consent: m('I understand'),
+    h: m<{ org: string }>('Unsubscribe from {org}'),
+    info: m(
+      'After you unsubscribe you will no longer receive mass email from this organization. You may still receive reminders and other email sent specifically to you as part of work you do in the organization.'
```

Comment:
maybe "...receive mass emails..." and "...and other emails..." with emails in plural?

Reply:
This is a great question! I'm not sure which one I prefer. I think both are "correct" and to me the "mass email" one sound better because it's not about specific emails, but about the concept of mass email. I gess it's like the difference between "mail" and "letters", where the former is the concept of mail, while the latter denotes physical objects. I think I'd like to keep it as is!